packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -44,6 +43,7 @@ interface InitializationData {
 }
 
 const ContractNameExtractorRegex = new RegExp(/(.*)Proxy/)
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
probably irrelevant but I'm curious, why you deleted the import? To get rid of the test folder in ts?